comp/core/workloadmeta/collectors/internal/remote/processcollector/process_collector.go

```diff
@@ -145,6 +145,10 @@ func (s *streamHandler) Port() int {
 	return s.port
 }
 
+func (s *streamHandler) Address() string {
+	return fmt.Sprintf(":%d", s.Port())
```

Comment:
Should we make the `Port` function private? Since it is no longer part of the generic interface

Reply:
:+1: 
Done